test/gtest/ucp/ucp_test.h

```diff
@@ -241,15 +242,18 @@ class ucp_test : public ucp_test_base,
     unsigned progress(int worker_index = 0) const;
     void short_progress_loop(int worker_index = 0) const;
     void flush_ep(const entity &e, int worker_index = 0, int ep_index = 0);
-    void flush_worker(const entity &e, int worker_index = 0);
+    void flush_worker(const entity &e, int worker_index = 0,
+                      const std::vector<entity*> &entities = {});
     void flush_workers();
     void disconnect(entity& entity);
     void check_events(const std::vector<entity*> &entities, bool wakeup,
                       int worker_index = 0);
     ucs_status_t
     request_progress(void *req, const std::vector<entity*> &entities,
                      double timeout = 10.0, int worker_index = 0);
-    ucs_status_t request_wait(void *req, int worker_index = 0, bool wakeup = false);
+    ucs_status_t request_wait(void *req, int worker_index = 0,
```

Comment:
minor: maybe change the order of parameters in this method, so that we could omit specifying `worker_index` and `wakeup` in most of the cases?
So it would be 
`request_wait(request, {e});`   instead of
`request_wait(request, 0, false, {e});`

Reply:
Actually the first PR pushed was like your suggestion, but to address review comment, I moved the `{e}` parameter at the end of the function. I am ok with the two options, but I guess we should keep it like this now?

Reply:
i did not notice that, but looks like less code is needed with the original version
@yosefe, wdyt?

Reply:
I'm ok with it as long as we have default value {}

Reply:
fixed it and kept default valud `{}`